package.json

```diff
@@ -17,32 +16,19 @@
     "lint-fix": "./node_modules/.bin/eslint --fix lib app",
     "dev": "NODE_ENV=dev && npm run build && electron app/main/index.js",
     "prod": "NODE_ENV=prod && npm run build && electron app/main/index.js",
-    "dev-down":
-      "node node_modules/db-migrate/bin/db-migrate down -e dev --config config/database.json",
-    "prod-down":
-      "node node_modules/db-migrate/bin/db-migrate down -e prod --config config/database.json",
-    "test-down":
-      "node node_modules/db-migrate/bin/db-migrate down -e test --config config/database.json",
-    "dev-up":
-      "node node_modules/db-migrate/bin/db-migrate up -e dev --config config/database.json",
-    "prod-up":
-      "node node_modules/db-migrate/bin/db-migrate up -e prod --config config/database.json",
-    "test-up":
-      "node node_modules/db-migrate/bin/db-migrate up -e test --config config/database.json",
-    "dev-reset":
-      "node node_modules/db-migrate/bin/db-migrate reset -e dev --config config/database.json",
-    "prod-reset":
-      "node node_modules/db-migrate/bin/db-migrate reset -e prod --config config/database.json",
-    "test-reset":
-      "node node_modules/db-migrate/bin/db-migrate reset -e test --config config/database.json",
-    "create-db-config":
-      "NODE_ENV=dev babel-node lib/io/sqlite/scripts/createDatabaseConfig.js",
-    "dev-windows":
-      "set NODE_ENV=dev&& npm run build && electron app/main/index.js",
-    "prod-windows":
-      "set NODE_ENV=prod&& npm run build && electron app/main/index.js",
-    "fmt":
-      "./node_modules/.bin/prettier --write '!(coverage|scripts|.vscode|node_modules)/**/!(.compiled|build|dist)/*.js'",
+    "dev-down": "node node_modules/db-migrate/bin/db-migrate down -e dev --config config/database.json",
```

Comment:
When i saved all these commands, they were formatted to multi-line by the linter. I guess they were not for you. How can we fix this inconsistency?

Reply:
Hmm. I think what's happening is `npm install --save some-module` formats the `package.json` file differently than [prettier](https://github.com/prettier/prettier) does. 

We could look into ignoring certain files, or adding a `.vscode` config to the project.